flink-python/docs/pyflink.batch.rst

```diff
@@ -0,0 +1,28 @@
+.. ################################################################################
+     Licensed to the Apache Software Foundation (ASF) under one
+     or more contributor license agreements.  See the NOTICE file
+     distributed with this work for additional information
+     regarding copyright ownership.  The ASF licenses this file
+     to you under the Apache License, Version 2.0 (the
+     "License"); you may not use this file except in compliance
+     with the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+    limitations under the License.
+   ################################################################################
+
+pyflink.batch package
+=====================
+
+Module contents
+---------------
+
+.. automodule:: pyflink.batch
```

Comment:
The package name of our Python API should look as follows:
- pyflink.common
- pyflink.table
- pyflink.datastream
- pyflink.dataset

So , I think the modue name of python api should be `pyflink.batch ` - > `pyflink.datastream`